lightning/src/routing/network_graph.rs

```diff
@@ -1658,4 +1747,132 @@ mod tests {
 		network.write(&mut w).unwrap();
 		assert!(<NetworkGraph>::read(&mut ::std::io::Cursor::new(&w.0)).unwrap() == *network);
 	}
+
+	#[test]
+	fn network_graph_score_payment() {
```

Comment:
Test is pretty good ! You can add whitespace before new comment line, it's cheap.

You can harden test to exerce MPP with success-failure, success-success, failure-failure cases.